src/libs/ajax.js

```diff
@@ -58,13 +58,13 @@ const fetchOk = (...args) => {
 
 
 const Sam = {
-  token: (namespace) => {
+  token: Utils.memoizeWithTimeout((namespace) => {
```

Comment:
Should this be more specific? Maybe namespaced to this file? I wonder about collisions.

Reply:
I'm wrong. Nevermind. I see the cache is private to the usage.